.ci_support/build_notebooks_damask.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+for f in $(cat .ci_support/exclude_damask); do 
```

Comment:
```suggestion
for f in $(cat .ci_support/exclude_fenics); do 
```

Reply:
Sorry I understood it other way around

Reply:
Yes... I anticipated that this could be misunderstood. My thought was that all files with endings "{-, _}damask" belongs to the damask testing, similarly for fenics.